### DISCUSSION TITLE
Requested VAA not found in store (transfer of weUSDC from SOL to ETH stuck)

### DISCUSSION THREAD

**Discussion by quentinadam** [wormhole-foundation/wormhole#505] [Problems / bugs] [answered]
Hello,

I tried to bridge 2,455,279 wormhole-v2-wrapped-USDC from Solana to Ethereum.
Here is the transaction on Solana : https://explorer.solana.com/tx/32bF7GLyizT6yzWsydHqPMznUra1oZJBXJde4yh7whZb5SWdtsm9nhqyTVv94cvfNJgwCgqsbeQCnc5XMeZLrWJ3.

The Solana transaction went through (2,455,279 wormhole-v2-wrapped-USDC were burned) but when I try to recover the USDC on Ethereum, I am getting a "requested VAA not found in store" error.

![Screenshot 2021-09-30 at 03 51 00](https://user-images.githubusercontent.com/2349972/135372574-36361d56-9aea-431b-9622-eca8ba37240f.png)

Could you please check?

**Comment by SEJeff** [comment]
@quentinadam  Steve is trying to steal your wallet private key or seed phrase.

The wormhole project will **NEVER** ask for your private key or seed phrase. If you give it out, they will steal your funds. The official wormhole page is https://wormholebridge.com/#/

**Comment by leoluk** [comment, marked as answer]
Please try the recovery flow again - the network recently made an effort to retry a handful of observations.

**Reply by ndkazu** [reply]
hello, I am experiencing the exact same problem, I am bridging Dai from polygon to Avalanche.
the tx hash is
0xf16d90597775d7f8d06c1cc6f9ec40592bb592b3627d5a60ea47dd31ae5a0ed8

**Reply by blablalf** [reply]
I am also experiencing the same problem with a transfer of eth from aurora to bsc weth, tx hash is 0x15237acf747e10576ec604738768f9da1ffc19b8298a70f27048fe08c33f1b43

**Reply by 0xoluwa** [reply]
me too.........help pls

0x3fdeaaf1748bf4e5353accba5f9302d10ae0895ff449538a2a537f5d67d163bc

**Reply by blablalf** [reply]
Don't worry, you need to wait 24 to 48h to be able to request your transaction manually

**Reply by lyonthreads** [reply]
I am also experiencing the same problem with a transfer of eth from sol to eth, tx hash is 3HPUyEDB71cmBE2ir4MDRxjLaeazaHNgo67iSt6KYSq1Jx1p9skwNAN92JUAUtb87EpWVi4hqg8GPTLLjDa32upm

**Comment by Proadik** [comment]
any updates here? same here https://solscan.io/tx/2mHFU7nUXmSn2ike2GxP2Yyvxmzg1qdQf4aSmhDrCrYzGBiinGe9EhvF3i5KaTSGgT1xQZkjJWd7h9zDJAmjosqD

**Reply by blablalf** [reply]
https://github.com/wormhole-foundation/wormhole/discussions/505#discussioncomment-3678268

**Comment by evan-gray** [comment]
All of the transactions referenced in this thread have been redeemed. Locking this discussion. 🙂
